### DISCUSSION TITLE
rFSMTools:  A set of tools to build, debug and execute lua-based state machines.

### DISCUSSION THREAD

**Discussion by apaikan** [robotology/community#212] [Show and tell]
![rfsmtools_promo](https://cloud.githubusercontent.com/assets/4293456/24900632/c883d230-1ea4-11e7-9151-56f9ff1c274c.png)

### What is [rFSMTools](https://robotology.github.io/rfsmTools/)?
- rFSMTools is a set of C++ library (librFSM) to load and execute rFSM LUA-based state machines and a graphical modelling tools (rfsmGui) to create, debug and execute the state machine.

### Designed for developers 
- rFSM is a small and powerful Statechart implementation written in Lua for coordination of complex systems. Take maximum advantage of rFSM functionality using rfsmTools to accelerate your application deveploment, debugging and delivery.

### Easy to use
- Build and edit your state machine directly from rfsmGui modeling tool or any source code editor. We also offer an embedded state-machine source editor for editing and debugging state machines at run time.

### Few Dependencies 
- rFSMTools have been developed with minimum dependencies. The librFSM exploits Lua 5.x and rFSM (embeddable) lua package. The Gui simply relies on Qt 5.x and graphviz library for automatic layouting.

### Download 
- [rFSMTools website](https://robotology.github.io/rfsmTools/)
- [rFSMTools on github](https://github.com/robotology/rfsmTools)

**Comment by claudiofantacci** [comment]
Congratulation for the release! It will be of real help!
